vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/view/boutiquesParsing/BoutiquesInputFlag.java

```diff
@@ -0,0 +1,79 @@
+package fr.insalyon.creatis.vip.application.client.view.boutiquesParsing;
+
+import com.google.gwt.json.client.JSONObject;
+import fr.insalyon.creatis.vip.application.client.view.launch.FlagInputLayout;
+import fr.insalyon.creatis.vip.application.client.view.launch.LaunchFormLayout;
+
+import java.util.Map;
+
+/**
+ * Representation of an Flag input in an application Boutiques descriptor
+ *
+ * @author Guillaume Vanel
+ * @version %I%, %G%
+ */
+public class BoutiquesInputFlag extends BoutiquesInput{
```

Comment:
It would sound more natural, I guess the same is true for Number, String and NonFlag. I did it this way only so that they are displayed together and next to BoutiquesInput in IntellJ Project Panel (or any other file browser sorted alphabetically), for ease of use.